CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # nf-core/tools: Changelog
 
+# v2.9dev
+
```

Comment:
Maybe we should add a highlights section here: new download for tower function, nf-validation in every pipeline, min. Nextflow bump, what else?

Reply:
Would you rather add it here or in the release notes?

Reply:
Regardless of the place, I think the topics you mention are the most remarkable ones 👍

Reply:
Forgot how we usually do it 🙃
Release notes it is 🙂